src/config.js.template

```diff
@@ -3,12 +3,16 @@ var config = {
     accessToken: 'YOUR_ACCESS_TOKEN',
     showMarkers: true,
     markerColor: '#3FB1CE',
-    theme: 'light',
-    use3dTerrain: false,
+    //projection: 'equirectangular',
+    //Read more about available projections here
+    //https://docs.mapbox.com/mapbox-gl-js/example/projections/
+    inset: true,
+    theme: 'dark',
+    use3dTerrain: false, //set true for enabling 3D maps.
     title: 'The Title Text of this Story',
     subtitle: 'A descriptive and interesting subtitle to draw in the reader',
     byline: 'By a Digital Storyteller',
-    footer: 'Source: source citations, etc.',
+    footer: 'Source: source citations, etc. <br> Created using <a href="https://github.com/mapbox/storytelling">Mapbox Storytelling</a> template.',
```

Comment:
Might make sense to link to a new tab with `target="_blank"` in the `<a>` element